guides/v15-mdc-migration.md

```diff
@@ -345,6 +347,7 @@ TODO(mmalerba): link to density docs once they exist.
   instead.
 
 ### Input
+<!-- TODO(devversion): review -->
 
 * MatInput must be inside `<mat-form-field>`. Previously it was (unintentionally) possible to use an
   `<input matInput>` without the form field if the page loaded form field styles elsewhere.
```

Comment:
I don't know where this comes from- but it makes sense and allows to clean up some more stuff in the future.